meetings/membership-criteria.md

```diff
@@ -0,0 +1,64 @@
+# Membership criteria
+
+When you apply for membership, we review your application based on the [membership participation types](membership-participation-types.md) defined. We ask for some basic information about you. 
+
+## Basic information
```

Comment:
In this section, make it clear that applicants can (but are not required) to fill in activities in multiple sections.